python/requirements/CI-tests-pip/requirements.txt

```diff
@@ -3,7 +3,6 @@ pytest==6.2.2
 pytest-xdist==2.2.1
 jsonschema==3.2.0
 h5py==3.1.0
```

Comment:
Looks like there's still a h5py in here?

Reply:
Yes, there needs to be one. Somehow we had two!!

Reply:
Yep, my bad